modules/weko-itemtypes-ui/weko_itemtypes_ui/views.py

```diff
@@ -135,6 +137,16 @@ def register(item_type_id=0):
 def custom_property(property_id=0):
     """Renders an primitive property view."""
     lists = ItemTypeProps.get_records([])
+
```

Comment:
Please update below source code:
from:
```
   if billing_perm and billing_perm.is_active:
        pass
    else:
        for prop in props:
            if prop.id == WEKO_BILLING_FILE_PROP_ID:
                props.remove(prop)

```
to:
```
   if not billing_perm or not billing_perm.is_active:
        for prop in props:
            if prop.id == WEKO_BILLING_FILE_PROP_ID:
                props.remove(prop)
```

Reply:
Thank for your comment.
It had been fix at last rev.